src/nwp_consumer/internal/config/config.py

```diff
@@ -49,6 +49,7 @@ class S3Config(_EnvParseMixin):
     """Config for S3."""
 
     AWS_S3_BUCKET: str
-    AWS_ACCESS_KEY: str
-    AWS_ACCESS_SECRET: str
     AWS_REGION: str
+    # Optional, attempts to use default AWS credentials if not set
+    AWS_ACCESS_KEY: str | None = None
```

Comment:
I tried something like that, do you also need to adjust the `_EnvParseMixin`

Reply:
I can write a test for this is you like, make sure when you make a `S3Config` it stays as None

Reply:
I think envparrse mixin handles it via the `getattrs`?

Reply:
Yeah feel free to add in a test!

Reply:
Ah I see yeah, will need to change the default value casting behaviour a bit.

Reply:
yea ill do that

Reply:
Sorry to have come round to your original suggestion only after going through the other options!

Reply:
no worries, yea I think thats why i did it orginally, 

shall I give it go on this branch?

Reply:
Please do :)